docker/compose/public_full_node/docker-compose.yaml

```diff
@@ -31,7 +31,7 @@
 version: "3.8"
 services:
     fullnode:
-        image: libra/validator:testnet
+        image: libra/validator:master_609c0b05
```

Comment:
I might argue that pegging these all on testnet is probably the right way to do it, in the future, we'll have more mobility in that space. Alternatively we could leverage devnet and do that for all of them. That way we don't get pegged on a version that must be manually updated.

Reply:
any thoughts on this one?

Reply:
I have it set to `devnet` as `testnet` doesn't exist in either libra/diem